### PR TITLE
feat(vpc): improve get private-network to show all attached resources

### PR DESCRIPTION
**Pull request by yfodil**

### CODE REVIEW

File: internal/namespaces/vpc/v1/custom_private_network.go [hunk 1]
```diff
@@ -49,18 +84,185 @@ func privateNetworkGetBuilder(c *core.Command) *core.Command {
 			}
 		}
 
+		// Baremetal
+		baremtalPNAPI := baremetal.NewPrivateNetworkAPI(client)
```
Comment (remyleone) [inline internal/namespaces/vpc/v1/custom_private_network.go:88]: Could you extract all those methods?

File: internal/namespaces/vpc/v1/custom_private_network.go [hunk 2]
```diff
@@ -4,19 +4,53 @@ import (
...
-	type customServer struct {
+	type customInstanceServer struct {
 		ID         string               `json:"id"`
 		Name       string               `json:"name"`
 		State      instance.ServerState `json:"state"`
 		NicID      string               `json:"nic_id"`
 		MacAddress string               `json:"mac"`
 	}
+	type customBaremetalServer struct {
+		ID                 string                 `json:"id"`
```
Comment (remyleone) [inline internal/namespaces/vpc/v1/custom_private_network.go:26]: Could you add the IP address within the private network?

File: internal/namespaces/vpc/v1/custom_private_network.go [hunk 3]
```diff
@@ -49,18 +84,185 @@ func privateNetworkGetBuilder(c *core.Command) *core.Command {
...
+			customGateways,
 		}, nil
 	}
 
 	c.View = &core.View{
 		Sections: []*core.ViewSection{
-			{FieldName: "Servers", Title: "Servers"},
+			{
+				FieldName: "InstanceServers",
+				Title:     "InstanceServers",
```
Comment (remyleone) [inline internal/namespaces/vpc/v1/custom_private_network.go:244]:
```suggestion
				Title:     "Instance Servers",
```

File: internal/namespaces/vpc/v1/custom_private_network.go [hunk 4]
```diff
@@ -49,18 +84,185 @@ func privateNetworkGetBuilder(c *core.Command) *core.Command {
...
+			},
+			{
+				FieldName: "BaremetalServers",
+				Title:     "BaremetalServers",
```
Comment (remyleone) [inline internal/namespaces/vpc/v1/custom_private_network.go:248]:
```suggestion
				Title:     "Baremetal Servers",
```

File: internal/namespaces/vpc/v1/custom_private_network.go [hunk 5]
```diff
@@ -49,18 +84,185 @@ func privateNetworkGetBuilder(c *core.Command) *core.Command {
...
+			},
+			{
+				FieldName: "LBs",
+				Title:     "LBs",
```
Comment (remyleone) [inline internal/namespaces/vpc/v1/custom_private_network.go:252]:
```suggestion
				Title:     "Load-Balancers",
```

File: internal/namespaces/vpc/v1/custom_private_network.go [hunk 6]
```diff
@@ -49,18 +84,185 @@ func privateNetworkGetBuilder(c *core.Command) *core.Command {
...
+			},
+			{
+				FieldName: "RdbInstances",
+				Title:     "RdbInstances",
```
Comment (remyleone) [inline internal/namespaces/vpc/v1/custom_private_network.go:256]:
```suggestion
				Title:     "Rdb Instances",
```

File: internal/namespaces/vpc/v1/custom_private_network.go [hunk 7]
```diff
@@ -49,18 +84,185 @@ func privateNetworkGetBuilder(c *core.Command) *core.Command {
...
+			},
+			{
+				FieldName: "RedisClusters",
+				Title:     "RedisClusters",
```
Comment (remyleone) [inline internal/namespaces/vpc/v1/custom_private_network.go:260]:
```suggestion
				Title:     "Redis Clusters",
```

File: internal/namespaces/vpc/v1/custom_private_network.go [hunk 8]
```diff
@@ -49,18 +84,185 @@ func privateNetworkGetBuilder(c *core.Command) *core.Command {
...
+			},
+			{
+				FieldName: "Gateways",
+				Title:     "Gateways",
```
Comment (remyleone) [inline internal/namespaces/vpc/v1/custom_private_network.go:264]:
```suggestion
				Title:     "Public Gateways",
```

File: internal/namespaces/vpc/v1/custom_private_network.go (hunk 2 above)
Comment (yfodil) [reply internal/namespaces/vpc/v1/custom_private_network.go]: It's not possible to get the IP address, I added the Vlan number instead

File: internal/namespaces/vpc/v1/testdata/test-get-private-network-simple.golden [hunk 9]
```diff
@@ -1,34 +1,59 @@
...
+ID                                    NAME                    STATE     NIC ID                                MAC ADDRESS
+89040064-4dd1-414d-9f5a-6239a0055847  cli-srv-epic-goldstine  archived  8c08dd03-da4d-4e44-8a6a-564d6df9ae13  02:00:00:12:e9:f2
+
+Baremetal Servers:
+ID   NAME  STATE  BAREMETAL NETWORK ID  VLAN
+
+Load-Balancers:
+ID   NAME  STATE
+
+Rdb Instances:
```
Comment (remyleone) [inline internal/namespaces/vpc/v1/testdata/test-get-private-network-simple.golden:23]: In case a ressource is missing could you avoid showing it up?

**Pull request merged**